Scripts/fn_switch.sqf

```diff
@@ -0,0 +1,53 @@
+#include "..\script_component.hpp"
+
+/*
+ * Author: Tyrone
+ * Functionality for toggling switches, When "OFF" switch is red and power levels are 0, when "ON" switch is Green and power levels are either 1 or player defined.
+ *
+ * Requires fn_switchAction.sqf
```

Comment:
Why? This function could easily add actions as well, you already have require the object as argument.